content/docs/common-areas/caching.md

```diff
@@ -1,65 +1,62 @@
 
 Cache invalidation depends upon the following factors:
 
-1. Time: The cached content is automatically invalidated when it reaches its expiration time. Drupal Cache API can be used to manually alter or set the desired expiration time.
-2. Context: The invalidation can also be handled based upon certain conditions apart from the expiration time. For ex. you might want to invalidate the cache based upon the type of user visiting the site.
-3. Dependencies: Cache invalidation may also depend upon number of other factors such as - changes made to the existing content of a specific type, curation of a listing block that lists the content tagged with a specific category and many more.
+- **Time**: Cache content is automatically invalidated when it expires. Drupal's Cache API allows manual adjustment of expiration times.
+- **Context**: Invalidation can also occur based on conditions apart from expiration time such as user type.
+- **Dependencies**: Cache invalidation can be triggered by changes to content types, specific categories, or the curation of listing blocks, among other factors.
 
 ## Cache API in Drupal
 
-Drupal with the help of CACHE API provides a way to manually store or cache data based upon the following factors - time, context and dependencies. These factors are called the cacheability metadata. The API consists of the following properties:
+Drupal's [Cache API](https://api.drupal.org/api/drupal/core%21core.api.php/group/cache/) manages data storage based on factors like time, context, and dependencies, known as Cacheability Metadata. Key properties include:
 
-1. Cache max-age: This property can be used to set the duration for which the content should be cached.
-2. Cache contexts: For dependencies on the request context.
-3. Cache tags: Tags can used to manage the dependencies on data managed by Drupal, i.e. type of content, category, user role etc.
+- **Cache Max-Age**: Sets content caching duration.
+- **Cache Contexts**: Manages dependencies based on request context.
+- **Cache Tags**: Handles dependencies on Drupal-managed data like content types or categories.
 
 ### Cache Bins
 
-In Drupal, the cache storage is separated into bins. These bins are used to store cached data of different types. Some of the common cache bins are as follows:
+In Drupal, cache storage is organized into bins, each storing different data types. Common bins include **bootstrap**, **render**, **data**, and **discovery**.
 
-1. bootstrap: Data needed from the beginning to the end of most requests, that has a very strict limit on variations and is invalidated rarely.
-2. render: Contains cached HTML strings like cached pages and blocks, can grow to large size.
-3. data: Contains data that can vary by path or similar context.
-4. discovery: Contains cached discovery data for things such as plugins, views_data, or YAML discovered data such as library info.
+Custom cache bins can be created by defining them in the `modulename.services.yml` file.
```

Comment:
Removing the description of common bins and then mentioning this detail seems unnecessary. Let's just say that modules may create cache bins for their own use.

Reply:
Also, there is inconsistency between newlines. Here, the next line is rendered in the same paragraph but it is on a separate line in markdown. We don't use newlines for this in the rest of the handbook.

Reply:
> Removing the description of common bins and then mentioning this detail seems unnecessary. Let's just say that modules may create cache bins for their own use.

@hussainweb: I shared the link below that line, and the description of each common bin is there.

Reply:
I don't see the description. See https://deploy-preview-205--axelerant-engg-handbook.netlify.app/docs/common-areas/caching/#cache-bins.